playground/playground.go

```diff
@@ -13,17 +13,45 @@ import (
 	"github.com/gopherjs/gopherjs/compiler"
 	"github.com/gopherjs/gopherjs/js"
 	"github.com/neelance/go-angularjs"
+	"honnef.co/go/js/dom"
+	"honnef.co/go/js/xhr"
 )
 
 type Line map[string]string
 
 var output []Line
 
+//const snippetStoreHost = "localhost:8080"
+const snippetStoreHost = "gotools.org:26204" // TODO: Decide on where the snippet-store server will be hosted.
```

Comment:
For example; although I'm not sure why you need the custom port, a frontend proxy could do the right thing per-domain (proxy gotools.org:80 to your current thing, proxy snippets.gotools.org:80 to the snippet service)